scss/_patterns_lists.scss

```diff
@@ -186,15 +187,21 @@ $spv-list-item--inner: null;
           width: $bullet-width;
         }
       }
+    }
 
-      & + .p-stepped-list__content {
-        @media (max-width: $breakpoint-heading-threshold) {
-          margin-left: $bullet-width-mobile + $sph-inner;
-        }
+    h#{$i}.p-stepped-list__title + .p-stepped-list__content {
+      @media (max-width: $breakpoint-heading-threshold) {
+        margin-left: $bullet-width-mobile + $sph-inner;
+      }
 
-        @media (min-width: $breakpoint-heading-threshold) {
-          margin-left: $bullet-width + $sph-inner;
-        }
+      @media (min-width: $breakpoint-heading-threshold) {
+        margin-left: $bullet-width + $sph-inner;
+      }
+    }
+
+    .p-stepped-list--detailed .p-stepped-list__title + .p-stepped-list__content {
```

Comment:
I think this part doesn't need to be inside `for` loop, it's unnecessarily repeated in output CSS:

<img width="687" alt="Screenshot 2020-05-18 at 13 34 13" src="https://user-images.githubusercontent.com/83575/82209048-eb074180-990c-11ea-9fc6-6a3d83783282.png">